pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -2,26 +2,28 @@
 
 ### Deposit to Subaccount
 
+The below cmd allows depositing funds into a subaccount from a sender. This can be a self deposit with the sender and recipient address being the same
 ```bash
 dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
 ```
 
 Example:
```

Comment:
we should note that this is a self-deposit

Reply:
this is self explanatory based on the params?

Reply:
Added in the first snippet